src/Avalonia.Base/Metadata/XmlnsPrefixAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Avalonia.Metadata
+{
+    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+    public sealed class XmlnsPrefixAttribute : Attribute
```

Comment:
I guess we should also include class level comment to make it clear what this attribute does for the users.

Reply:
Added class level comment